integrations/disco/routes/post-credential-issue/client.ts

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+
+import { IssuedCredentials } from '../../utils/types'
+
+export async function appDiscoPostCredentialIssue(values: { recipientDid: any }) {
+  try {
+    const payload = {
+      schemaUrl: 'https://raw.githubusercontent.com/discoxyz/disco-schemas/main/json/ProofOfHackCredential/1-0-0.json',
+      subjectData: { ...values },
+      recipientDID: values?.recipientDid,
+    }
+    const { data }: { data: IssuedCredentials } = await axios.post(`/api/disco/credential-issue`, JSON.stringify(payload), {
```

Comment:
you can use `useQuery` from `tanstack-query` to handle the async state management

Reply:
for these i am currently using usemutate for statemanagement

Reply:
you're right